cmd/xgql/main.go

```diff
@@ -231,30 +243,22 @@ func main() {
 	kingpin.FatalIfError(h.ListenAndServe(), "cannot serve insecure HTTP")
```

Comment:
Was going to note something about handling graceful shutdown, but looks like we are already not performing it with existing servers. May be worth adding as a follow up 👍🏻

Reply:
>Was going to note something about handling graceful shutdown, but looks like we are already not performing it with existing servers. May be worth adding as a follow up 👍🏻

I started going down that path 😄. It ended up being a pretty large scope creep, hence the move back to something pretty simple.